go.mod

```diff
@@ -105,6 +105,8 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/huin/goupnp v1.0.0 // indirect
+	github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go v1.2.3-alpha.1 // indirect
```

Comment:
why does this keep reappearing? and sometimes it gets deleted?

Reply:
glitches, maybe